cms/lib/xblock/tagging/tagging.py

```diff
@@ -43,7 +43,8 @@ def student_view_aside(self, block, context):  # pylint: disable=unused-argument
         Display the tag selector with specific categories and allowed values,
         depending on the context.
         """
-        if isinstance(block, CapaModule):
```

Comment:
I tried to change this view but it seems that in case of `author_view` aside block doesn't displayed at all. It seems that problems are specially displayed with a `student_view` in studio  preview mode:
![xblock_view_in_studio](https://cloud.githubusercontent.com/assets/1876893/14710036/74c35268-07dc-11e6-9dbc-0317ead78b12.jpeg)

There is a special [PreviewModuleSystem class](https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/preview.py#L99) that is used in CMS. And there is a special param:
```
    # xmodules can check for this attribute during rendering to determine if
    # they are being rendered for preview (i.e. in Studio)
    is_author_mode = True
```

This behavior could be changed to display questions with an `author_view`. And I've made attempts to do this. For example this one: https://github.com/edx/edx-platform/commit/a3f1a9648eb8d57fc0a57821c96a70f5097ca177
but I'm not sure that it is a very good solution